src/utils/helpers.js

```diff
@@ -30,3 +30,13 @@ export function solveMapping(mapping, props, context) {
 
   return mapping;
 }
+
+/**
+ * Determines if the given tree is a Baobab tree.
+ * FIXME: if Baobab ever implements something like Array.isArray we should use
+ * that instead of relying in the internal _identity = '[object Baobab]' value.
+ * See https://github.com/Yomguithereal/baobab/blob/master/src/baobab.js#L111
+ */
+export function isBaobabTree(tree) {
+    return !!(tree && tree.toString() === '[object Baobab]');
```

Comment:
If you want to be paranoid, you should also check that `tree.toString` is a function.